src/components/document/student-group-view.tsx

```diff
@@ -67,9 +71,13 @@ export const StudentGroupView:React.FC<IProps> = ({ groupId, setGroupId }) => {
                             selected={group.id === selectedId}
               />
               { groupUsers.map(u => {
-                  const className = classNames("member-button", "in-student-group-view", u.context);
+                  const className = classNames("member-button", "in-student-group-view", u.context,
+                                                {focused: u.user.id === focusedGroupUser.id});
                 return (
-                  <div key={u.user.name} className={className} title={u.user.name}>{u.user.name}</div>
+                  <button key={u.user.name} className={className} title={u.user.name}
+                      onClick={()=>handleFocusedUserChange(u)}>
```

Comment:
I would change this to
```suggestion
                      onClick={()=>handleFocusedUserChange(u.user)}>
```

then change the parameter type for `handleFocusedUserChange` accordingly.